app/components/content/talk_to_us_component.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class="talk-to-us-content">
```

Comment:
I do wonder slightly about putting this top of page @EllieNodder @gazcarr given that chat / phone are closed for long periods? (Plus also whether this means people won't scroll down to see the services)

![image](https://user-images.githubusercontent.com/87642394/200257660-8cccdab8-4937-4c24-ad68-586178c3fa57.png)

Reply:
This conversation did come up during design. 

I think your point is valid but I also think its worth monitoring closely with a plan in place to review and decide on its position after X amount of time. 
 
 The tricky thing is measuring success on this page, it may well be that the majority of people looking for help might want the phone number, as far as I am aware we don't have anything to back this up though. We do know that people scroll, but obviously they won't if they are intent on finding the number. Maybe we start out with an A/B test trying the different positions, either way it probably needs a plan to discuss how we can best measure this.

Reply:
Thanks @gazcarr - agree that success is difficult on this page. Perhaps something to discuss with Andy H

@ethax-ross Just a thought - are we able to change the position of the modules based on the time of day or is that too complex?!

Reply:
We could change their position based on the time of day but we'd need to do it in Javascript as the page will be cached; we'd need to be careful not to cause a cumulative layout shift but I think it should be doable without too much effort